app/views/games/show.html.erb

```diff
@@ -18,4 +18,5 @@
   <% end %>
 </table>
 <br />
+<%= link_to 'Forfiet', forfeit_game_path, class: 'btn btn-danger', method: :PUT %>
```

Comment:
We can add a modal here that links to the Forfeit. We can also alter it to verify `if current_user != game.user` then `alert: 'You are not part of the game!`.